commnuity-calls/README.md

```diff
@@ -0,0 +1,21 @@
+# Flashbots Community Calls
+
+A Community Call is a flash organization of humans and robots devoted to exploring, designing, and synthesizing various aspects of stateful blockchains. The goal of a community call is to produce high-quality, relevant technical and engineering documents that influence the way people design, use, and implement blockchain protocols. These documents include design space discussion recordings, protocol standards, best practices, and various informational documents. This process is loosely inspired by the mission of the [IETF](https://www.ietf.org/about/introduction/#mission) and the [Ethereum Project Management](https://github.com/ethereum/pm) process.
+
+## ☀️ Active Community Calls
+
+| Name | Organizer       | Purpose   |  Cadence |  Calendar |
+|--------------------|-----------------|--------------------|------------------|-------------------|
+| [Flashwares](./flashwares/README.md)         | [SUAVE Andromeda]()     |  TEE Education and Design Discussions  | Burst Installments   | [Forum](https://collective.flashbots.net/tag/flashwares) |
+| [SUAVE.dev](./suapp-developer-streams/README.md)         | [cryptowanderer](https://x.com/cryptowanderer) & [Brock](https://x.com/zeroXbrock/)     | SUAPP Development Office Hours and Tutorials   |  Monthly  | [Community Calender](http://calendar.flashbots.net/) |
+| [All Core Devs](./All-Core-Devs/README.md)         | [dmarz](https://x.com/DistributedMarz) | Core Protocol Development Discussions   |  Monthly  | [Community Calender](http://calendar.flashbots.net/) |
+
+## 🧊 Retired Community Calls
+
+| Name | Organizer       | Purpose                     |
+|--------------------|-----------------|--------------------------------|
+| [MEV Roasts](./mev-roasts/README.md)         | [Flashbots](https://writings.flashbots.net/)     |  Communication bridge between Flashbots and the broader community   |
+
```

Comment:
Could be worth listing the abandoned whiteboarding sessions if only for inspo? Only one recording was made public though: https://collective.flashbots.net/t/whiteboard-an-overview-of-credible-commitment-devices/926

Reply:
if only 1 i'd rather not

Reply:
The only reason I suggest it is that it demonstrates that there are a variety of formats which could work well for community calls. No strong opinion though